consolidate/IO/deck.py

```diff
@@ -17,8 +17,11 @@ def __init__(self, inputhpath):
         
         
     def create_folder_structure(self):        
-        self.plot_dir = self.doc["Plot"]["folder"]
-        check_folder = os.path.isdir(self.plot_dir)
+        self.plot_dirTemp = self.doc["Plot"]["folder1"]
```

Comment:
See comment in deck.yaml about this name

Reply:
solved in #6fa6763